rt/src/test/java/io/fabric8/maven/rt/SpringbootConfigmapBoosterIT.java

```diff
@@ -35,6 +34,8 @@
 
     private final String SPRING_BOOT_CONFIGMAP_BOOSTER_GIT = "https://github.com/snowdrop/spring-boot-configmap-booster.git";
 
+    private final String RELEASED_VERSION_TAG = "tags/1.5.10-5";
```

Comment:
Couldn't we just put here the number part (`1.5.10-5`) and just append `tags/` inside the `setupSampleTestRepository`? So it is easier to track.